tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/Properties/AssemblyInfoPropertiesProviderTests.cs

```diff
@@ -183,6 +183,8 @@ public async Task ProjectFileProperties_GetEvaluatedPropertyAsync(string code, s
                     """[assembly: System.AssemblyDescriptionAttribute("MyDescription")]""")]
         public async Task SourceFileProperties_SetPropertyValueAsync(string code, string propertyName, string propertyValue, string expectedCode)
         {
+            using var _ = SynchronizationContextUtil.Suppress();
```

Comment:
Comment about why we need to do this?

Reply:
Adding comments to the `Suppress` method itself, as the reason is a bit long for a single-line comment here (and on other usages).